src/v3/src/transformer/layout/securityQuestion/securityQuestionVerify.ts

```diff
@@ -45,6 +45,8 @@ export const transformSecurityQuestionVerify: IdxStepTransformer = ({ transactio
       ? relatesTo?.value?.profile?.question as string
       : loc(`security.${relatesTo?.value?.profile?.questionKey}`, 'login'),
   }];
+  answerElement.options.inputMeta.secret = true;
```

Comment:
This shouldn't be needed, however, there is no harm doing this here so all good. 

Did you notice this using mocks or an actual org?

Reply:
While testing on my local.

Reply:
@chaitanyaachanta-okta this seems like a bug with the backend API. In general we shouldn't have to set this value on the frontend if it can be done on the backend. We can leave this here to keep it working properly for now but please file a JIRA for the home team to change this on the backend and link it here in a comment so we can remove it eventually.

Reply:
@shawyu-okta Sure, will do. We own the backend, will fix as part of that ticket.

Reply:
Thanks! Before merging, please add a comment above this line with a JIRA reference